lib/node/RequireChunkLoadingRuntimeModule.js

```diff
@@ -35,6 +36,32 @@ class RequireChunkLoadingRuntimeModule extends RuntimeModule {
 		const hasJsMatcher = compileBooleanMatcher(
 			chunkGraph.getChunkConditionMap(chunk, chunkHasJs)
 		);
+
+		let rootOutputDir = "'.'";
+		try {
+			if (chunk.hasEntryModule()) {
+				const outputName = this.compilation.hooks.assetPath.call(
```

Comment:
That's the part I'm most unsure about, how to get the destination of the entry relative to the output dir to calculate how much dirs we need to go up?